classes/helpers/NostoHelperCurrency.php

```diff
@@ -230,6 +242,40 @@ private static function createWithCldr(array $currency)
         );
     }
 
+    /**
+     * @param Context $context
+     * @param array $currency
+     * @return NostoSDKCurrencyFormat
+     * @throws NostoSDKException
+     * @throws \PrestaShop\PrestaShop\Core\Localization\Exception\LocalizationException
+     */
+    private static function createWithContextLocale(Context $context, array $currency) {
+        $locale = $context->getCurrentLocale();
+        $priceSpec = $locale->getPriceSpecification($currency[self::ISO_CODE_FIELD]);
+        $symbols = $priceSpec->getAllSymbols();
+        if (!isset($symbols[self::DEFAULT_SYMBOL_FIELD])) {
+            throw new NostoException(
+                sprintf(
+                    'Could not find default %s symbol to use in currency formatting',
+                    self::DEFAULT_SYMBOL_FIELD
+                )
+            );
+        }
+        $numberSymbolList = $symbols[self::DEFAULT_SYMBOL_FIELD];
+        $symbolPos = Tools::strpos($priceSpec->getPositivePattern(), self::CURRENCY_SYMBOL_MARKER);
+        $symbolPosition = $symbolPos === 0;
```

Comment:
I'd rename this var to clarify, like
```suggestion
        $isCurrencyBeforeAmount = $symbolPos === 0;
```

Reply:
Dude, thanks for the input, makes sense :)